server/planning/planning/planning.py

```diff
@@ -1831,7 +1831,28 @@ def _iter_recurring_plannings_to_update(self, updates, original, update_method):
     "description_text": metadata_schema["description_text"],
     "internal_note": {"type": "string", "nullable": True},
     "anpa_category": metadata_schema["anpa_category"],
-    "subject": metadata_schema["subject"],
+    "subject": {
+        "type": "list",
+        "mapping": {
+            "type": "object",
+            "dynamic": False,
+            "properties": {
+                "qcode": not_analyzed,
+                "name": {
+                    "type": "string",
```

Comment:
here the type should be `not_analyzed`

Reply:
I tried, but in this case, some testcases are failing :- like this [case](https://github.com/superdesk/superdesk-planning/blob/f82f39d39fac832892299790900505d2c5d7a694/server/features/search_events.feature#L156)

Reply:
in theory `"type": "keyword"` should be enoug